Project.toml

```
@@ -18,7 +18,7 @@ Unicode = "4ec0a83e-493e-50e2-b9ac-8f72acf5a8f5"
 [compat]
 CodeTracking = "0.5.9"
 JuliaInterpreter = "0.7.16"
-LoweredCodeUtils = "0.4"
+LoweredCodeUtils = "0.4, 1.1"
```

Why didn't CompatHelper do
```suggestion
LoweredCodeUtils = "0.4, 1"
```
instead?

---

@DilumAluthge ^

---

Originally this was an intentional feature.... not sure why. I've forgotten my rationale.

Want to make a pull request to change it?